modules/swagger-codegen/src/main/resources/csharp/Project.mustache

```diff
@@ -88,7 +88,10 @@
     {{/netStandard}}
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="**\*.cs"/>
+    <Compile Include="Api\**\*.cs"/>
```

Comment:
@jimschubert Yes, i know the namespace can be changed (as I did, if you have a look at the related issue). However, as I can tell, the namespace has no influence on folder structure. 

I am using both, a default namespaced ("IO.Swagger") and a custom namespaced ("MyProject.Api") generated project. Both got the same folder structure for me.